src/styles/app.css

```diff
@@ -0,0 +1,266 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: 'Montserrat', Arial, Helvetica, sans-serif;
+}
+
+/*
+Navbar Styles
+*/
+.header-container {
+  display: flex;
+  background-color: #fff;
+  padding: 1.813rem 6.25rem;
+  border-bottom: solid 1px #e8e8e8;
+}
+
+.main-navbar {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  column-gap: 2.938rem;
+  width: 100%;
+}
+
+.main-navbar h1 {
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: #0290ff;
+}
+
+.nav-links {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  list-style: none;
+  column-gap: 2.563rem;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+}
+
+.nav-link-a {
+  color: #121212;
+  text-decoration: none;
+  opacity: 0.65;
+}
+
+.nav-link-a:hover {
+  opacity: 0.85;
+}
+
+.nav-link-a:active {
+  color: #121212;
+  opacity: 1;
+}
+
+.icon-holder {
+  align-self: flex-end;
+  border: 1px solid #e8e8e8;
+  border-radius: 50%;
+}
+
+.nav-icon {
+  width: 1.063rem;
+  height: 1.063rem;
+  padding: 0.875rem;
+}
+
+/*
+Main Container Styles
+*/
+.main-container {
+  padding: 0 6.25rem;
+  background-color: #fafafa;
+}
+
+.book-list-container {
+  padding: 2.5rem 0;
+}
+
+/*
+Book Card Styles
+*/
+.book-card {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  background-color: #fff;
+  margin-top: 2.313rem;
+  padding: 2rem 1.688rem;
+  border: solid 1px #e8e8e8;
+  border-radius: 4px;
+}
+
+#book-category {
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+  opacity: 0.5;
+}
+
+#book-title {
+  font-size: 1.375rem;
+  font-weight: bold;
+  color: #121212;
+  margin: 0.25rem 0;
+}
```

Comment:
- [OPTIONAL] Great job styling your app, it looks great! I would suggest using `class` selectors instead of `id` selectors, this way it's easier to reuse them and this helps you to avoid duplicated code.